conf/modules/phasing.config

```diff
@@ -25,46 +25,42 @@ process {
     }
 
     withName: '.*:PHASING:HIPHASE_SNV' {
-
+        ext.prefix = { "$meta.id}_phased" }
         ext.args = { [
             '--ignore-read-groups',
-            "--stats-file ${meta.id}.stats.tsv",
-            "--blocks-file ${meta.id}.blocks.tsv",
-            "--summary-file ${meta.id}.summary.tsv"
+            "--stats-file ${meta.id}_phased.stats.tsv",
+            "--blocks-file ${meta.id}_phased.blocks.tsv",
+            "--summary-file ${meta.id}_phased.summary.tsv"
         ].join(' ') }
 
         publishDir = [
-            path: { "${params.outdir}/phasing/hiphase/snv/${meta.id}" },
+            path: { "${params.outdir}/" },
             mode: params.publish_dir_mode,
-            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+            saveAs: { filename -> filename.equals('versions.yml') ? null : ((filename.endsWith('bam') || filename.endsWith('bai')) ? "aligned_reads/${meta.id}/${filename}" : "phasing/hiphase/snv/${meta.id}/${filename}" ) }
         ]
     }
 
     withName: '.*:PHASING:HIPHASE_SV' {
-
+        ext.prefix = { "$meta.id}_phased" }
         ext.args = { [
             '--ignore-read-groups',
-            "--stats-file ${meta.id}.stats.tsv",
-            "--blocks-file ${meta.id}.blocks.tsv",
-            "--summary-file ${meta.id}.summary.tsv"
+            "--stats-file ${meta.id}_phased.stats.tsv",
+            "--blocks-file ${meta.id}_phased.blocks.tsv",
+            "--summary-file ${meta.id}_phased.summary.tsv"
         ].join(' ') }
-
         publishDir = [
-            path: { "${params.outdir}/phasing/hiphase/sv/${meta.id}" },
+            path: { "${params.outdir}/" },
             mode: params.publish_dir_mode,
-            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+            saveAs: { filename -> filename.equals('versions.yml') ? null : ((filename.endsWith('bam') || filename.endsWith('bai')) ? "aligned_reads/${meta.id}/${filename}" : "phasing/hiphase/sv/${meta.id}/${filename}" ) }
```

Comment:
Why put the bam bai in different directories?

Reply:
To have all aligned reads in `results/aligned_reads` regardless of if they are haplotagged or not. Otherwise, `results/aligned_reads` would not exist when phasing is active. 

There's currently three phasing options: WhatsHap, Hiphase with SNVs or HiPhase with SVs. WhatsHap is the default and tested option.

WhatsHap is split into two processes, `WHATSHAP_PHASE` and `WHATSHAP_HAPLOTAG`, where phase phases the VCF and haplotag haplotags the BAM files, so there it's easier to output the BAMs from `WHATSHAP_HAPLOTAG` to `results/aligned_reads` and the VCF from `WHATSHAP_PHASE` (currently) to `phasing/whatshap/phase`.

In HiPhase, phasing and haplotagging happens within the same process, but to make the output similar to that of WhatsHap, I decided to split the BAM and VCF files into separate directories. 

There's a problem with HiPhase though, you are forbidden to run ONT data with it. So it is likely that we change out HiPhase in favour of longphase, which has the same phase/haplotag structure as WhatsHap. 

There are some stuff to do later: 
- If HiPhase is kept, `HIPHASE_SV` and `HIPHASE_SNV` should just be combined into one process. 
- Currently the SNVs that goes into the annotation are not phased, but they should be. Then if the annotated SNVs are phased, the phased, unannotated SNVs do not really need to be duplicated in `results/phasing/whatshap/phase/`. That is also a reason why I'm removing the BAM files from `results/phasing/`, because I see that directory soon being empty.